openproblems/tasks/denoising/README.md

```diff
@@ -8,10 +8,11 @@ A key challenge in evaluating denoising methods is the general lack of a ground
 
 # The metrics
```

Comment:
Can you please follow the format in https://github.com/openproblems-bio/openproblems/blob/main/openproblems/tasks/label_projection/README.md#the-metrics ? Thanks!

Reply:
Ping @wes-lewis